helm/nessie/values.yaml

```diff
@@ -106,7 +107,7 @@ log:
     release: ~
     # -- Package prefixes that belong to your application.
     inAppPackages:
-    - org.projectnessie
+      - org.projectnessie
```

Comment:
Is this (and similar) white space change necessary?

Reply:
Ah my bad. That was my editor. I didn’t intend to commit that. Will fix when I get back to my desk.